akeneo/4-apache/Dockerfile

```diff
@@ -113,10 +113,10 @@ RUN curl -sS https://getcomposer.org/installer | \
 
 WORKDIR /var/www/html
 
-RUN curl -sSLO https://github.com/akeneo/pim-community-standard/archive/v$AKENEO_VERSION.tar.gz && \
-    tar -xzf v$AKENEO_VERSION.tar.gz -C . && \
-    rm v$AKENEO_VERSION.tar.gz && \
-    mv pim-community-standard-$AKENEO_VERSION pim-community-standard && \
+RUN curl -sSLO https://github.com/akeneo/pim-community-standard/archive/$AKENEO_VERSION.tar.gz && \
+    tar -xzf $AKENEO_VERSION.tar.gz -C . && \
+    rm $AKENEO_VERSION.tar.gz && \
+    mv pim-community-standard-"${AKENEO_VERSION:1}" pim-community-standard && \
```

Comment:
do you really need the quotes here ?
Seems different from akeneo5 Dockerfile
Default variable value will be "1" ?

Reply:
you're right, we don't need them :) 

nb: the ${:x} syntax is used in bash to remove the first X characters from a string (from v5.3 to 5.3)